openslides/motions/config_variables.py

```diff
@@ -237,4 +237,4 @@ def get_config_variables():
         label='Show paragraph numbering (only in PDF)',
```

Comment:
Here the same. By the way, check if this config is still necessary because we have line numbering now. @emanuelschuetze?